zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -171,7 +172,6 @@ public ZeppelinServer() throws Exception {
   public static void main(String[] args) throws InterruptedException {
 
     ZeppelinConfiguration conf = ZeppelinConfiguration.create();
-    conf.setProperty("args", args);
```

Comment:
Usage was removed in https://github.com/apache/zeppelin/commit/0330452231c9dae85bb642fbd7a8b9cc16c6777d#diff-4bd397174ee0f21a8d3b788b97bc8711L487